docs/src/simulation_tips.md

```diff
@@ -181,7 +182,7 @@ to achieve this
 
 - Use the [`nvidia-smi`](https://developer.nvidia.com/nvidia-system-management-interface) command
   line utility to monitor the memory usage of the GPU. It should tell you how much memory there is
-  on your GPU and how much of it you're using.
+  on your GPU and how much of it you're using. You can run it from Julia with the command ``run(`nvidia-smi`)``.
```

Comment:
These lines could be a bit more concise:

"The [`nvidia-smi`](https://developer.nvidia.com/nvidia-system-management-interface) utility monitors memory usage and GPU utilization. `nvidia-smi` can be launched from Julia by typing ``run(`nvidia-smi`)``."

Reply:
Fixed